Dockerfile

```diff
@@ -21,7 +24,7 @@ RUN mkdir /usr/tsunami/plugins \
 WORKDIR /usr/repos/tsunami-security-scanner
 COPY . .
 RUN ./gradlew shadowJar \
-    && cp $(find "./" -name 'tsunami-main-*-cli.jar') /usr/tsunami/tsunami.jar \
+    && cp "$(find "./" -name 'tsunami-main-*-cli.jar')" /usr/tsunami/tsunami.jar \
```

Comment:
nit: Can we update the inner double quote to single quote for readability purpose?

Reply:
I am not sure your change really does the job, I will check and if it was fine, I will apply it.

Reply:
Can you please write the exact command you mean? Because from your comment, if you mean `"$(find "./" -name "tsunami-main-*-cli.jar")"`, this will obviously not work, so I think I'm misunderstanding your suggestion.

Reply:
Tested this locally and it works.
```
set -x
cp "$(find "./" -name "tsunami-main-*-cli.jar")" /tmp/test2/tsunami3.jar 
++ find ./ -name 'tsunami-main-*-cli.jar'
```
Interesting point for me is when executing this after `set -x` the command becomes `find ./ -name 'tsunami-main-*-cli.jar'`.

But it works anyway and has been commited in https://github.com/google/tsunami-security-scanner/pull/89/commits/824a5df982a9d1544cf2233837572aa3aaab99aa, thanks!